herders/templates/herders/profile/data_logs/dashboard.html

```diff
@@ -5,7 +5,7 @@
 {% endblock %}
 
 {% block logs %}
-    <h1>Personal Data Logs</h1>
+    <h1>Personal Data Logs{% if total %} (<a class="data-logs-detach" style="cursor: pointer;">Detach</a>){% endif %}</h1>
```

Comment:
Consider using a bootstrap button just below the header: https://getbootstrap.com/docs/3.3/css/#buttons-tags